.github/pull_request_template.md

```diff
@@ -4,38 +4,57 @@
   If a section of the PR template does not apply to this PR, then delete that section.
  -->
 
-## Motivation
+## What type of PR is this?
 
-_(fill-in or delete this section)_
+_(REQUIRED)_
+
+<!--
+  /kind bug
+  /kind cleanup  
+  /kind documentation
+  /kind feature
+ -->
```

Comment:
Do we really want to use these prow commands here? We don't have prow available on this repository so we have to apply the labels manually. From a user perspective I would now expect that something happens after the PR got created, like prow comes along and labels the PR accordingly like in k8s.

What if we transform it to a clickable list:
```suggestion
- [ ] bug
- [ ] cleanup
- [ ] documentation
- [ ] feature
```
And still apply the labels manually afterwards?

Reply:
Fixed per your suggestion.